lib/Onnxifi/Base.cpp

```diff
@@ -157,8 +157,18 @@ onnxStatus Graph::setIOAndRun(uint32_t inputsCount,
     }
 
     if (inOnnxTensorSize > inPhPtr->getType()->size()) {
+      std::stringstream ss;
+      for (const auto i : inOnnxTensorDims) {
+        ss << i << ", ";
```

Comment:
Would be nicer without the trailing comma, too bad we don't have folly::join :-)

Reply:
Python style. :)